order/buyer/issue/delivery-schedule.md

```diff
@@ -0,0 +1,68 @@
+---
+description: >-
+  Choose between the standard or simple delivery schedule
+---
+
+# Delivery schedule
+
+## Standard versus simple delivery schedule
+
+Tradecloud works with a delivery schedule per order line.
+Each delivery line in a schedule consists of a delivery date and a quantity.
+
+Some ERP systems like SAP work natively with delivery schedule's.
+Use the Tradecloud **standard** delivery schedule in this case.
+
+Other ERP systems can only work with one date and one quantity per order line.
+Use the Tradecloud **simple** delivery schedule in this case.
+
+## Standard delivery schedule
+
+Use the field `lines.deliverySchedule` for the requested planned delivery schedule of this order line. 
+
+Provide at least one or multiple delivery schedule lines. The total number of delivery schedule lines is limited to 100 lines per order line.
+
+### `deliverySchedule` fields
+
+* `position`: the position in the delivery schedule. Not to be confused with the `lines.position`.
+* `date`: the requested delivery date of this delivery schedule position. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
+* `quantity`: the requested quantity of this delivery schedule position. Quantity has a decimal `1234.56` format with any number of digits.
+* `status`: The [logistics status](#logistics-status) of this delivery line according to the buyer.
+* `transportMode`: The Mode of Transport used for the delivery of goods as required by the buyer. UNECE.org Recommendation 19 is advised for Mode of Transport values.
+
+{% hint style="warning" %}
+`position` must be unique within the delivery schedule and never change. Never renumber or re-use a `position`.
+
+When no `position` is provided, most delivery schedule features are disabled.
```

Comment:
Perhaps it's good what is disabled, or perhaps what _isn't_ disabled

Reply:
@marcmatt can we not make the position "mandatory" in the documentation?

Reply:
@roy-tc @Sjorstradecloud @maxim-md I also made it mandatory in the API specs
https://github.com/tradecloud/tradecloud-connectors-scala/pull/759/commits/7a5afcf12399f2037fd50f0120066febd56ded83 

@ystasiv-tradecloud this might break the accp tests, running here
https://github.com/tradecloud/tradecloud-connectors-scala/actions/runs/5157178640

Reply:
@Sjorstradecloud 
Making delivery schedule position mandatory broke 15 acceptance tests
So I reverted it, and for now we only say in the API manual and specs description it is mandatory.

If all buyers use positions, including Damen, then lets make it technical mandatory, including the API specs and update tests
Do we have a ticket to make the delivery schedule position mandatory in the buyer line in the API connector?